### DISCUSSION TITLE
Specify JIra API version

### DISCUSSION THREAD

**Discussion by tb582** [pycontribs/jira#1530] [Q&A]
How do you specify the API versioning that the issues= command sends ? ie 

https://<site>/rest/api/2/search?
vs /api/latest/search ... 

?

**Comment by adehad** [comment]
Hi @tb582 .

Sorry for the delay, you would be looking to override:
https://jira.readthedocs.io/api.html#jira.client.JIRA.DEFAULT_OPTIONS


specifically the `'rest_api_version'` key. The override should be passed in as a dictionary to the `options` argument to the JIRA client.
